sample/common/build.gradle

```diff
@@ -56,6 +56,11 @@ kotlin {
     }
 
     sourceSets {
+        all {
+            languageSettings {
+                useExperimentalAnnotation("com.russhwolf.settings.ExperimentalSettingsApi")
```

Comment:
trikot.viewmodel itself doesn't use that lib. Is that really required ?

Reply:
Without it, I get this error:

```
> Task :common:linkDebugFrameworkIos
w: Opt-in requirement marker com.russhwolf.settings.ExperimentalSettingsApi is unresolved. Please make sure it's present in the module dependencies
```

Reply:
Is it an external project that fails ? Or simply the library compilation ? I really do not get why as we don't use multiplatform-settings in trikot.viewmodels